engine/src/scriptrunner.h

```diff
@@ -199,24 +224,45 @@ public slots:
      */
     int random(int minTime, int maxTime);
 
+protected slots:
+    /** Triggered when the script's execution pauses to await the starting of a function */
+    void slotWaitFunctionStarted(quint32 fid);
+
+    /** Triggered when the script's execution pauses to await the completion of a function */
+    void slotWaitFunctionStopped(quint32 fid);
+
 protected:
     /** QThread reimplemented method */
     void run();
 
 private:
+    /** ScriptRunner function operations enum to handle start/stop/wait commands */
+    enum class SRFuncOpe : char
```

Comment:
This is not how enums are defined in this project. See
https://github.com/mcallegari/qlcplus/blob/master/engine/src/chaser.h#L217
and please name it FunctionOperation

Reply:
Fixed on commit f3d3ca0d0fae741279f2ad6434bf0eed902cd16b